iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -34,12 +34,66 @@ public static DigitalTwinClient CreateFromConnectionString(string connectionStri
             return new DigitalTwinClient(iotHubConnectionString.HttpsEndpoint, sharedAccessKeyCredential, handlers);
         }
 
+
         private DigitalTwinClient(Uri uri, IotServiceClientCredentials credentials, params DelegatingHandler[] handlers)
```

Comment:
Unlike in previous iterations, this iteration fixes the stale DNS bug for existing users. They do not need to change constructors because we still only have one, and it has the bug fix